src/app/projects/create-project/create-project.styl

```diff
@@ -87,6 +87,24 @@ md-select.create-project-select
   text-align center
   line-height 40px
 
+.create-project-panel-progress-item-number-in-progress
+  animation: pulse 5s infinite;
+
+@keyframes pulse {
+  0% {
+    background-color white
+    color black
+  }
+  50% {
+    background-color $item-enabled-color-bg
+    color white
+  }
+  100% {
+    background-color white
+    color black
+  }
+}
+
```

Comment:
we do not need webkit prefix (or mozilla, etc). It's now in HTML5/CSS standard and it appears in Safari